<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationConfigurationDialog</class>
 <widget class="QDialog" name="StationConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Station Configuration</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Lon:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lonDegEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>deg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lonMinEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lonDirEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>N/S</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="calcButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Calc</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Station Symbol</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Group/overlay</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="groupEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Symbol</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="symbolEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="symbolSelectButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Power / Height / Gain / Directivity</string>
   </property>
   <widget class="QCheckBox" name="disablePHGCheckbox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>334</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Power:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="powerCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>0 Watts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 Watts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 Watts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9 Watts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16 Watts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25 Watts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>36 Watts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>49 Watts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64 Watts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>81 Watts</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="heightCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>10 Feet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20 Feet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40 Feet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>80 Feet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>160 Feet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>320 Feet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>640 Feet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1280 Feet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2560 Feet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5120 Feet</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>331</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Gain:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gainCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>0 dB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 dB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 dB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 dB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 dB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 dB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 dB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7 dB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8 dB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9 dB</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Directivity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="directivityCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Omni</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>45°</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>90°</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>135°</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>180°</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>225°</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>270°</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>315°</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>360°</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>63</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Lat:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="latDegEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>deg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="latMinEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="latDirEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>N/S</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Callsign:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="callsignEdit"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="compressedPacketsCheckbox">
      <property name="text">
       <string>Send Compressed Posits</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>257</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Position Ambiguity:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.11 Mile</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.15 Miles</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11.51 Miles</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>69.09 Miles</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>431</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Comment:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="commentLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StationConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StationConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
